aspnetcore/blazor/debug.md

```diff
@@ -45,54 +44,117 @@ Debugging requires either of the following browsers:
 * Google Chrome (version 70 or later)
 * Microsoft Edge preview ([Edge Dev Channel](https://www.microsoftedgeinsider.com))
 
+## Enable debugging
+
+To enable debugging in an existing Blazor WebAssembly app, update *launchSettings.json* in the startup project of your app. Set `useWebAssemlbyDebugging` to `true` and include the following `inspectUri` property in each launch profile:
```

Comment:
Turns out `useWebAssemblyDebugging` isn't necessary and shouldn't be in the template at all.

```suggestion
To enable debugging in an existing Blazor WebAssembly app in Visual Studio or Visual Studio Code, update *launchSettings.json* in the startup project of your app to include the following `inspectUri` property in each launch profile:
```